glossary.md

```diff
@@ -23,6 +23,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | algorithm's performance   | chất lượng của thuật toán                                      |               |
 | avoidable bias            | độ chệch tránh được                                            |               |
 | artificial data synthesis | tổng hợp dữ liệu nhân tạo                                      |               |
+| attention mechanisms | attention mechanisms                                                |               |
```

Comment:
Từ này e nghĩ có thể dịch thành "Cơ chế attention", từ khá phổ biến và dễ hơn cho người đọc.

Reply:
Từ attention mình thấy không có nghĩa gì đặc biệt trong ML, tác giả của paper đó cũng là cố tình mượn từ "attention" để ví von kĩ thuật của họ cho hay lên.
Vậy nên mình đề xuất dịch hết tất cả sang tiếng việt.
```suggestion
| attention mechanisms | cơ chế chú ý.                                                              |               |
```
```suggestion
| attention mechanisms | kỹ thuật chú ý                                                             |               |
```

Reply:
Em nghĩ vẫn nên giữ lại từ attention nguyên bản và dịch từ mechanism, vì khi đọc về phần này và muốn search để tìm hiểu kĩ hơn, nếu sử dụng cụm từ "kỹ thuật/cơ chế chú ý" sẽ hơi lạ và khó tìm, em đề xuất cụm từ "cơ chế/ kỹ thuật attention" vì nó khá phổ biến và hầu hết các bài viết tiếng việt đều sử dụng cụm từ này ạ.

Reply:
lý do này mình thấy chưa thuyết phục :))
Nếu vì để dễ tìm mà mình không dịch thì những từ như convolution, recurrent thì sao nhỉ?
Mình đợi thêm các ý kiến khác xem nhé.

Reply:
Mình cũng chưa thuyết phục về lý do giữ lại, nhưng dịch ra hiện tại mình thấy nó sát nghĩa rồi nhưng có gì đó còn gợn, không biết còn cách dịch nào khác nữa không. 

Về việc tra cứu mình đã làm Bảng thuật ngữ, đây là cái mình làm ra dùng để tra cứu rồi nên việc giữ lại mình thấy không quá cần thiết, có chăng thì ghi chú để người đọc lần đầu có thể hình dung nhanh thôi chứ không cần thiết để lại.

Reply:
Mình có thể mở ngoặc từ gốc ở những lần đầu gặp một từ. Có thể mở ngoặc nhiều lần nếu thấy cần thiết.

Từ "mechanism" dịch là "cơ chế". "attention" có thể là "chú ý" hoặc "tập trung".

Reply:
> Nếu vì để dễ tìm mà mình không dịch thì những từ như convolution, recurrent thì sao nhỉ?
> Mình đợi thêm các ý kiến khác xem nhé.

Từ Attention hiện tại khá phổ thông, mình nghĩ ai dùng hoặc làm DL cũng nghe qua nó vì quá nổi. Khác với các từ khác như convolution hay recurrent đã được dịch và dùng từ trước, từ này chưa được dịch tới mãi gần đây. 

Dịch Attention là "tập trung" thì có vẻ không ổn, vì nghĩa nó khác với "Focus". . 
Dịch là "chú ý" thì nghe nó hơi ... sao sao, chưa hay lắm. 

Hiện tại mình thiên về việc giữ nguyên từ khóa này hơn. Để rồi sau dần xem có từ nào hay không.

Reply:
Từ attention đã trở thành từ khóa rồi, chứ không chỉ là từ không thường nữa.

Reply:
Ok vậy mình làm cái vote nhé:
- "cơ chế chú ý" 👍 
- "kỹ thuật chú ý" 😄
- "cơ chế tập trung"  🚀 
- "cơ chế attention" 🎉

Reply:
Em thấy cơ chế chú ý cũng được, lần đầu dịch có lẽ mở ngoặc như anh Tiệp bảo

Reply:
@ngcthuong : regularization là từ khóa và cũng được dịch ra là "điều chuẩn", sao lần này dịch "attention" bác lại chùn bước vậy 💃 :D

Reply:
Chưa thấy từ đó "hay" đó :))). Nghe từ "chú ý" nó cứ quê quê sao ấy.

Reply:
Còn regularization tớ làm với nó từ đầu 2012 nên nắm rõ hơn. Từ đó bắt nguồn từ bên NLP nên không chắc ăn lắm :)). Còn nghĩa tiếng Việt theo từ thì chuẩn rồi.

Reply:
em google "điều chuẩn" thì toàn ra liên quan đến hạt điều 💃 . Nói vui thế, chứ khi đọc sách thì nó có context, người đọc chắc không bị lost; mình dịch thuật ngữ nếu đảm bảo được khi đọc nên hiểu được ý nó nói gì và không bị ambiguous thì là ok.

Reply:
"đệp" mới chịu cơ :)))

Reply:
Vậy có lẽ em dịch tạm thành "cơ chế chú ý" ạ, nếu có ý tưởng gì thì sẽ sửa sau

Reply:
@william-vu , nay ngồi đọc và dịch thử các đoạn liên quan tới từ khóa attention mechanism (không phải ở cuốn này), thấy dịch là "cơ chế chú ý" nghe khá là ngượng mồm khi đọc cả đoạn văn. Để một mình, ngữ nghĩa thì quá ok. 

Giờ mình lại thấy từ khóa "cơ chế tập trung" nghe hay hơn. Vì sau khi mình kiểm tra thêm cũng thấy Attention dùng có tác dụng, hoặc mục đích "focus" vào các features quan trọng.  

Cứ để sau dịch thêm xem thế nào.

Reply:
lần đầu nhìn "cơ chế tập trung" mình lại nghĩ đến "centralized mechanism" :D (chắc tại bitcoin đang hot, nên chỉ nhớ đến centralized vs decentralized)

Reply:
Em vừa dịch thử theo "cơ chế chú ý" ở bản gần nhất, đúng là nghe ngượng thật ạ :D